ccd-definition/CaseEventToComplexTypes/HearingRequirementsDJ-SDO-HNL-nonprod.json

```diff
@@ -0,0 +1,30 @@
+[
```

Comment:
is this one part of the SDO or would it just be DJ HNL file name?

Reply:
this is part of SDO